src/TreeNode.js

```diff
@@ -20,14 +20,6 @@ export default class TreeNode {
      * @constructor
      */
     constructor(options) {
-        // variables for the node
-        this.left = undefined;
-        this.right = undefined;
-        this.distribution = undefined;
-        this.splitValue = undefined;
-        this.splitColumn = undefined;
-        this.gain = undefined;
-
         // options parameters
         this.options = options;
```

Comment:
I used this because I create the tree nodes recursively, maybe I can access to the options using this.options instead of saving those inside the object, what do you think @maasencioh?